src/Stratis.FederatedPeg.Features.FederationGateway/TargetChain/CrossChainDBTransaction.cs

```diff
@@ -0,0 +1,264 @@
+﻿using System;
+using System.Collections.Generic;
+using DBreeze.DataTypes;
+using NBitcoin;
+using Stratis.Bitcoin.Utilities;
+using Stratis.FederatedPeg.Features.FederationGateway.Interfaces;
+
+namespace Stratis.FederatedPeg.Features.FederationGateway.TargetChain
+{
+    /// <summary>Supported DBreeze transaction modes.</summary>
+    public enum CrossChainTransactionMode
```

Comment:
If I correctly understand and we are only talking about transaction in the DB sense of the term, I think the name is a bit ambiguous here: CrossChainTransactionMode could be easily interpreted as different modes of performing a Cross chain transfer no?

Reply:
Changes to `CrossChainDBTransactionMode`.